### ISSUE TITLE
gitpod fails on build

### ISSUE THREAD

**Issue by TrevorJTClarke** [near-examples/wallet-example#138]
```
yarn dev
yarn run v1.22.4
$ yarn build && yarn dev:deploy:contract && parcel src/index.html
$ node asconfig.js
 asconfig.js is deprecated use `asb`to build files.  NOTE: output now in ./build/release/ 
Error: Cannot find module '/workspace/wallet-example/asconfig.json'
Require stack:
- /workspace/wallet-example/node_modules/asbuild/dist/cli.js
- /workspace/wallet-example/node_modules/asbuild/dist/index.js
- /workspace/wallet-example/node_modules/asbuild/index.js
- /workspace/wallet-example/node_modules/near-sdk-bindgen/compiler.js
- /workspace/wallet-example/node_modules/near-sdk-as/compiler.js
- /workspace/wallet-example/asconfig.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at getConfig (/workspace/wallet-example/node_modules/asbuild/dist/cli.js:105:16)
    at main (/workspace/wallet-example/node_modules/asbuild/dist/cli.js:76:18)
    at module.exports.compile (/workspace/wallet-example/node_modules/near-sdk-bindgen/compiler.js:42:3)
    at Object.<anonymous> (/workspace/wallet-example/asconfig.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/workspace/wallet-example/node_modules/asbuild/dist/cli.js',
    '/workspace/wallet-example/node_modules/asbuild/dist/index.js',
    '/workspace/wallet-example/node_modules/asbuild/index.js',
    '/workspace/wallet-example/node_modules/near-sdk-bindgen/compiler.js',
    '/workspace/wallet-example/node_modules/near-sdk-as/compiler.js',
    '/workspace/wallet-example/asconfig.js'
  ]
}
Error: Cannot find module '/workspace/wallet-example/asconfig.json'
Require stack:
- /workspace/wallet-example/node_modules/asbuild/dist/cli.js
- /workspace/wallet-example/node_modules/asbuild/dist/index.js
- /workspace/wallet-example/node_modules/asbuild/index.js
- /workspace/wallet-example/node_modules/near-sdk-bindgen/compiler.js
- /workspace/wallet-example/node_modules/near-sdk-as/compiler.js
- /workspace/wallet-example/asconfig.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at getConfig (/workspace/wallet-example/node_modules/asbuild/dist/cli.js:105:16)
    at compileProject (/workspace/wallet-example/node_modules/asbuild/dist/cli.js:123:18)
    at main (/workspace/wallet-example/node_modules/asbuild/dist/cli.js:92:9)
    at module.exports.compile (/workspace/wallet-example/node_modules/near-sdk-bindgen/compiler.js:42:3)
    at Object.<anonymous> (/workspace/wallet-example/asconfig.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1158:30) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/workspace/wallet-example/node_modules/asbuild/dist/cli.js',
    '/workspace/wallet-example/node_modules/asbuild/dist/index.js',
    '/workspace/wallet-example/node_modules/asbuild/index.js',
    '/workspace/wallet-example/node_modules/near-sdk-bindgen/compiler.js',
    '/workspace/wallet-example/node_modules/near-sdk-as/compiler.js',
    '/workspace/wallet-example/asconfig.js'
  ]
}
asc assembly/main.ts --debug --measure --validate --target release --binaryFile build/release/main.wasm
WARNING Unknown option '--validate'
I/O Read   :     7.892 ms  n=307
I/O Write  :     0.644 ms  n=1
Parse      :   867.968 ms  n=133
Initialize :    45.641 ms  n=1
Compile    :   353.772 ms  n=1
Emit       :   249.910 ms  n=1
Validate   :   116.563 ms  n=1
Optimize   :     1.447 ms  n=1
Transform  :          n/a  n=0
$ near dev-deploy && echo "# This file used by parcel
CONTRACT_NAME=$(cat neardev/dev-account)" > .env.local
Starting deployment. Account id: dev-1605737769477-6090440, node: https://rpc.testnet.near.org, helper: https://helper.testnet.near.org, file: ./out/main.wasm
An error occured
[Error: ENOENT: no such file or directory, open './out/main.wasm'] {
  [stack]: "Error: ENOENT: no such file or directory, open './out/main.wasm'",
  [message]: "ENOENT: no such file or directory, open './out/main.wasm'",
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './out/main.wasm'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by jilt** [comment]
I solved this by copying and renaming the wasm from the folder it's compiled in to the .out/main.wasm (created the folder myself), looks like a bug on the main near api js, many devs found that while starting to build on near examples

**Comment by bucanero** [comment]
This should be fixed now (#208)
